exo-run/src/app-runner.ls

```diff
@@ -39,6 +54,16 @@ class AppRunner extends EventEmitter
     DockerCompose.kill-all-containers {cwd: @docker-config-location, @write}, -> process.exit exit-code
 
 
+  _compile-online-text: ->
+    online-texts = {}
+    for protection-level of @app-config.services
+      for role, service-data of @app-config.services[protection-level]
+        if service-data.location #TODO: compile online text for external services
```

Comment:
Yes, but unfortunately it's blocked on this: https://github.com/Originate/exosphere-sdk/issues/278

I've made sure to track this in this ticket: https://github.com/Originate/exosphere-sdk/issues/274